lib/matplotlib/text.py

```diff
@@ -367,7 +367,7 @@ def _get_layout(self, renderer):
         of a rotated text when necessary.
         """
         thisx, thisy = 0.0, 0.0
-        lines = self.get_text().split("\n")  # Ensures lines is not empty.
+        lines = self._get_wrapped_text().split("\n")  # Ensures lines is not empty.
```

Comment:
Do you understand what the comment is about?

Reply:
Not completely, though I can see that there are a bunch of necessary variables defined in the loop 383-414 which would not exist if `lines` was empty.  So maybe this is just providing reassurance for that?